infra/project-config/system-notifications.tf

```
@@ -0,0 +1,17 @@
+locals {
+  topics = {
+    "workflows" = {
+    }
+  }
+
+  system_notifications_config = {
+    channels = {
+      "workflow-failures" = {
```

```suggestion
      workflow-failures = {
```

I don't think either `-` or `_` in variable names requires quotes?